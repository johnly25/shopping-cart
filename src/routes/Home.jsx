import { Box } from "@chakra-ui/react";
import background from '../assets/homepage_background.jpg'

function Home() {
    return (
        <>
            <Box h='100vh' bgImage={background} bgSize='cover'>
            </Box>
        </>
    )
}

export default Home;