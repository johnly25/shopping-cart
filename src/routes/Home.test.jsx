import { render, screen } from "@testing-library/react";
import { beforeEach, describe } from "vitest";
import Home from "./Home";

describe('snapshops for routes', ()=> {
    test("renders home", ()=> {
      const {container} = render(<Home/>);
      expect(container).toMatchSnapshot();
    });
});