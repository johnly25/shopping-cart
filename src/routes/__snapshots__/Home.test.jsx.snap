// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshops for routes > renders home 1`] = `
<div>
  <div
    class="css-rwzxts"
  />
</div>
`;
